client/tests/test_mimc.py

```diff
@@ -2,17 +2,40 @@
 #
 # SPDX-License-Identifier: LGPL-3.0+
 
-from zeth.core.mimc import MiMC7
+from zeth.core.mimc import MiMC7, MiMC31
 from unittest import TestCase
 
 
 class TestMiMC(TestCase):
 
-    @staticmethod
-    def test_mimc_round() -> None:
+    def test_mimc7_round(self) -> None:
         m = MiMC7("Clearmatics")
         x = 340282366920938463463374607431768211456
         k = 28948022309329048855892746252171976963317496166410141009864396001978282409983  # noqa
         c = 14220067918847996031108144435763672811050758065945364308986253046354060608451  # noqa
-        assert m.mimc_round(x, k, c) == \
-            7970444205539657036866618419973693567765196138501849736587140180515018751924  # noqa
+        self.assertEqual(
+            m.mimc_round(x, k, c),
+            7970444205539657036866618419973693567765196138501849736587140180515018751924)  # noqa
+
+    def test_mimc7(self) -> None:
+        m = 28948022309329048855892746252171976963317496166410141009864396001978282409983  # noqa
+        k = 14220067918847996031108144435763672811050758065945364308986253046354060608451  # noqa
+        h = 14404914332179247771191118015445305957789480573634910846417052002923707343766  # noqa
+        result = MiMC7().hash(m, k)
+        self.assertEqual(h, result)
+
+    def test_mimc31_round(self) -> None:
+        m = MiMC31()
+        x = 340282366920938463463374607431768211456
+        k = 3614637061043937583146271435827337369189798160947949526058695634226054692860  # noqa
+        c = 5775606169419625606859319496982126279674858730791300481051019590436651369410  # noqa
+        h = 5523634951166384704739554074217840169048851347397743343350526776025419511991  # noqa
+        result = m.mimc_round(x, k, c)
+        self.assertEqual(h, result)
```

Comment:
May be worth to avoid the use of one letter names for clarity (c = round_constant, k = key, m = mimc, x = message (msg), h =  expected_res)

Reply:
renamed variables